s17/regexp.rb

```diff
@@ -0,0 +1,18 @@
+puts //.class
+
+s = 'The Ruby Programming Language is amazing'
+puts s =~ /y/
+puts /y/ =~ s
+p s =~ /x/   #=> nil
+puts s =~ /Ruby/ # position R
+
+
```

Comment:
Please don't leave all of these empty lines at the end of the file. Try to write cleaner code.

Reply:
Sorry, this was because the Linux keyboard did not receive the sign ( ~ ) and I commented it at the end of the file.